packages/bot-web-ui/src/components/quick-strategy/inputs/qs-input.tsx

```diff
@@ -58,11 +65,10 @@ const QSInput: React.FC<TQSInput> = observer(
                                                         const value = Number(field.value) - 1;
                                                         handleChange(e, String(value % 1 ? value.toFixed(2) : value));
                                                     }}
-                                                    disabled={disabled}
                                                 >
                                                     -
                                                 </button>
-                                            ) : null
+                                            ) : undefined
```

Comment:
to fix TS errors:
`Type 'Element | null' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>> | undefined'.`

`  Type 'null' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>> | undefined'.`

The error occurs because TypeScript cannot determine that the element that returns the conditional expression is a React element. To solve this problem, you can explicitly specify the type of the returned element